specs/linux/syslog.table

```diff
@@ -0,0 +1,10 @@
+table_name("syslog")
+schema([
+    Column("time", INTEGER),
+    Column("host", TEXT),
+    Column("severity", INTEGER),
+    Column("facility", TEXT),
+    Column("tag", TEXT),
+    Column("message", TEXT),
+])
```

Comment:
If you add an `attributes` the generated documentation will include an icon indicating this is a subscriber-based table: https://github.com/facebook/osquery/blob/master/specs/linux/user_events.table#L14